mantidproject/mantid
# Add ISIS SANS GUI to Workbench

**Description of work.**
Moves the ISIS SANS v2 GUI to workbench.
This required exposure of `JobTreeView` and `Hint` C++ classes (among others) to Python.
The icons used in the SANS GUI was changed to use font awesome, bringing it inline with the rest of workbench

**Report to:** nobody

**To test:**
Open up MantidPlot and go through the testing script as provided in https://github.com/mantidproject/mantid/blob/master/dev-docs/source/Testing/SANSGUI/SANSGUITests.rst

Open up Workbench and run through the same script. 

Everything should work as described in the script for both MantidPlot and Workbench


Fixes #24669 
Fixes #24724 


*This does not require release notes* because **SANS 4.0/ workbench does not yet have release notes**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
